src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,50 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, V> {
 
+    private T firstElement;
+    private V secondElement;
+
+    private Pair(T t, V v) {
+        firstElement = t;
+        secondElement = v;
+    }
```

Comment:
```suggestion
    private Pair(T firstElement, V secondElement) {
        this.firstElement = firstElement;
        this.secondElement = secondElement;
    }
```
in constructor, it's better to use the same naming